webpack.config.js

```diff
@@ -103,10 +103,6 @@ const config = {
 
 if (process.env.NODE_ENV === 'development') {
   config.mode = 'development';
-  config.devtool = 'source-map';
-  config.entry = ['webpack-dev-server/client?http://localhost:4242'].concat(
-    config.entry
-  );
```

Comment:
I took the opportunity to clean this up. Indeed the devtool 'source-map' is already set in the main config, and I'm not sure what this entry is for, at least this seems to work fine without it.